tests/test_items.py

```diff
@@ -769,12 +769,18 @@ def test_metadata():
 
     -   If the generic Metadata object is assigned as input, it gets converted
         into an object of the right, more specific metadata class.
+
+    Request templates are an exception.
```

Comment:
I think that can be fine to incude meta to the request template as well.

Reply:
For the meta, I'd keep both probability and dateDownloaded. 

Ity may be useful even if we use https://github.com/TeamHG-Memex/Formasaurus to generate the templates, as we'd have e.g. a probability of a search form being a search form, and probability of search input being a search input.